react-native/realm.json

```diff
@@ -0,0 +1,4 @@
+{
+  "appId": "<Your App ID>",
+  "url": "http://localhost:8080"
```

Comment:
I think it should be "https://realm.mongodb.com", but it's probably ok to leave it off entirely for now. The support is there, so that's good.

Reply:
Sounds good, removed